docs/playbook/ingesting-data/005-ingesting-dynamo-db-tables.md

```diff
@@ -0,0 +1,177 @@
+      "--number_of_workers" = local.number_of_workers_for_mtfh_ingestion
+    }
+
+    crawler_details = {
+      database_name      = aws_glue_catalog_database.mtfh_landing_zone.name
+      s3_target_location = "s3://${module.landing_zone.bucket_id}/mtfh/"
+      table_prefix       = "mtfh_"
+      configuration = jsonencode({
+        Version = 1.0
+        Grouping = {
+          TableLevelConfiguration = 3
+        }
+      })
+    }
+  }
+  ```
+This is example terraform to create a glue job in the Data Platform account.
+You will need to changes the following values
+   - The module name - Declared at the top, in the example it is set to "ingest_dataset_tables". You should rename this to reflect the entire collection of data that is being ingested by this job.
+   - `job_name` - Name of your job with `${local.short_identifier_prefix}` prefixed in front of it like the example.
+   - `job_description` (Optional) - An optional description for what this job is doing.
+   - `database_name` - The name of the glue catalogue database where the tables will be written to in the data platform.
+   If you want to create the tables in the landing zone database, this will be `aws_glue_catalog_database.landing_zone_catalog_database.name`.
+   If you want to create the tables in a departments raw zone database, this will be `module.department_DEPARTMENT_NAME.raw_zone_catalog_database_name`.
+   - `--table_names` - A comma delimited list of table names to ingest from Dynamo DB.
+   - `--role_arn` - This should be the value of the data object created in the step above, for the example given it would be `data.aws_ssm_parameter.role_arn_access_housing_tables.value`.
+   This signifies to terraform to go and get the value of this parameter, in which you have saved the ARN of the role created in the initial step.
+   Using the data object instead to writing the actual role ARN here allows you to configure a different role ARN for each environment.
+   - `--number_of_workers` and `number_of_workers_for_glue_job` - Both of these values should be set to the same value. More workers will mean more paralellisation and so if there is a large amount of data you might want to increase this, but if there is a small amount of data it will be expensive to have this too high as it won't utilise all of the workers.
+   As a guide, we have a table that has around 90000 rows in it and we're using 12 workers, please speak to an engineer if you are unsure.
+
+   The following parameters are inside the `crawler_details`.
+   - `s3_target_location` and `--s3_target` - This will specify the location in S3 bucket that the data is written to.
+   The glue job will create a folder for each table that it is ingesting so you just need to give the prefixes for these tables to live in.
+   For example if you are writing to the landing this could be, `s3://${module.landing_zone.bucket_id}/mtfh/`, if the tables were coming from modern tools for housing.
```

Comment:
```suggestion
   For example if you are writing to the landing zone this could be, `s3://${module.landing_zone.bucket_id}/mtfh/`, if the tables were coming from modern tools for housing.
```